src/components/[guild]/LeaveButton/hooks/useLeaveGuild.ts

```diff
@@ -25,7 +23,7 @@ const useLeaveGuild = () => {
         status: "success",
       })
 
-      mutate((prev) => prev.filter(({ guildId }) => guildId !== id), {
+      mutate(undefined, {
```

Comment:
should't we just `mutate()` here?

Reply:
I'm not sure, the idea here was, that since we left the guild, we don't have any memberships anymore, so we just delete the data by mutating it to undefined